### ISSUE TITLE
ECMASCRIPT6 in SIMPLE_OPTIMIZATIONS

### ISSUE THREAD

**Issue by ralder** [monai/cc-web-runner#1]
Hi!
When I try use es6 to es3 transpilation with SIMPLE_OPTIMIZATIONS it returns error:

```
"JSC_MISSING_ES6_EXTERNS"
 "description":"Missing externs definition for Symbol. Did you forget to include the ES6 externs?"
```

(gcc as jar works fine with default settings and transpiled this code)

Maybe do you have an idea why this error occurs? 

```
curl -X POST -H "Content-Type: application/json" -d '{
    "options": {
        "languageIn": "ECMASCRIPT6",
        "languageOut": "ECMASCRIPT3"
        },
  "optimizations": {
    "level": "SIMPLE_OPTIMIZATIONS"
  },
  "sources": [{
    "fileName": "bar.js",
    "code": "

function test(bigParam1, undefined) {
    if (bigParam1) {
        return 0;
    }

    return function(m) {
        let x = 0;
        if (m != undefined) {
            return `result is ${x}`;
        }
    };
} console.log(test(1));"
  }]
}
' "http://localhost:8081/compile?debug"

```

If change SIMPLE_OPTIMIZATIONS to ADVANCED_OPTIMIZATIONS it works without this error.

**Comment by monai** [comment]
Fixed in v1.0.7

**Comment by ralder** [comment]
Great! 👍  Thank you.
